src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
 function convertToObject(sourceString) {
-  const result = {};
+  const stylesObject = {};
 
   sourceString
     .split(';')
     .map(el => el.trim())
     .filter(el => el)
     .map(el => el.split(': '))
     .forEach(el => {
-      result[el[0].trimEnd()] = el[1].trimStart();
+      stylesObject[el[0].trimEnd()] = el[1].trimStart();
     });
 
-  return result;
```

Comment:
why you've deleted "result" name, seems like it's commited to be =) 
just a short and understandable var name

Reply:
I saw this in the [checklist](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md)
> 2. [CODE STYLE] - Avoid variable namings like array , object arr, obj, **result**. It will decrease code readability